src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -107,16 +100,16 @@ public bool TryCompleteAsync(SocketAsyncContext context)
                 return TryCompleteOrAbortAsync(context, abort: false);
             }
 
-            public void AbortAsync(SocketAsyncContext context)
+            public void AbortAsync()
             {
-                bool completed = TryCompleteOrAbortAsync(context, abort: true);
+                bool completed = TryCompleteOrAbortAsync(null, abort: true);
                 Debug.Assert(completed);
             }
 
             private bool TryCompleteOrAbortAsync(SocketAsyncContext context, bool abort)
             {
```

Comment:
Since you've changed `AbortAsync` to always pass `null` for the `context` parameter, how about a `Debug.Assert(context != null || abort)`?